assets/src/helpers/icon.tsx

```diff
@@ -215,6 +217,8 @@ export const SaveIcon = svgIcon(saveIconSvg)
 
 export const SearchIcon = svgIcon(searchIconSvg)
 
+export const SearchMapIcon = svgIcon(searchMapIconSvg)
```

Comment:
**suggestion:** it's probably worth adding this icon to the list in `icon.test.tsx`, even if I have my doubts about how useful those particular tests are.